packages/functions/src/dal/migrationScripts/ChangeAppointmentStatusMigration.ts

```diff
@@ -0,0 +1,39 @@
+import { getAllLatestAppointments, setAppointment } from "../AppointmentDataAccessLayer";
+import { AppointmentStatus } from "@zm-blood-components/common/src";
+const fs = require('fs');
+
+export async function addStatusForAppointments(){
+  const appointments = await getAllLatestAppointments();
+
+  return appointments.map((appointment) => {
+    let newStatus = AppointmentStatus.AVAILABLE;
+    const now = new Date();
+    if(appointment.donorId !== "")
+    {
+      if (appointment.donationStartTime.toDate() < now)
+        newStatus = AppointmentStatus.COMPLETED; // should this be completed?
+      else
+        newStatus = AppointmentStatus.BOOKED;
+    }
+    else
+    {
+      if(appointment.donationStartTime.toDate() < now)
+        newStatus = AppointmentStatus.NOSHOW
```

Comment:
This should be AVAILABLE, no?
I think noShow is when it is booked but the donor did not show up

Reply:
can an appointment be AVAILABLE if the appointment date has passed?

Reply:
Yes, this is how we identify appointments that were not booked